src/budget.rs

```diff
@@ -57,6 +60,19 @@ impl Budget {
         Budget::After(Condition::Signature(from), Payment { tokens, to })
     }
 
+    /// Create a budget that pays tokens` to `to` after being witnessed by 2x `from`s
+    pub fn new_dual_authorized_payment(
```

Comment:
You can call this `new_2_2_multisig_payment`.  This PR gets us so close to M-N Multisig that you can call this transaction just a specialization of it.